internal/edge/stack.go

```diff
@@ -53,18 +53,18 @@ const (
 	edgeStackStatusAcknowledged
 )
 
-type engineStatus int
+type engineType int
 
 const (
-	_ engineStatus = iota
-	engineStatusDockerStandalone
-	engineStatusDockerSwarm
-	engineStatusKubernetes
+	_ engineType = iota
+	engineTypeDockerStandalone
+	engineTypeDockerSwarm
+	engineTypeKubernetes
 )
 
 // StackManager represents a service for managing Edge stacks
 type StackManager struct {
-	engineStatus *engineStatus
+	engineType   *engineType
```

Comment:
why need pointer here?

Reply:
to be able to have a null value (engine is not set yet)

Reply:
It's an ok option if it needs to be a null value.

But personally I would like to explicit define the null value as a enum number. For example:
const (
	engineTypeNone engineType = iota
	engineTypeDockerStandalone
	engineTypeDockerSwarm
	engineTypeKubernetes
)

Reply:
done